docs/configs.md

```diff
@@ -51,6 +51,8 @@ Name | Description | Default Value
 <a name="python.memory.gpu.maxAllocFraction"></a>spark.rapids.python.memory.gpu.maxAllocFraction|The fraction of total GPU memory that limits the maximum size of the RMM pool for all the Python workers. It supposes to be less than (1 - $(spark.rapids.memory.gpu.maxAllocFraction)), since the executor will share the GPU with its owning Python workers. when setting to 0 it means no limit.|0.0
 <a name="python.memory.gpu.pooling.enabled"></a>spark.rapids.python.memory.gpu.pooling.enabled|Should RMM in Python workers act as a pooling allocator for GPU memory, or should it just pass through to CUDA memory allocation directly. When not specified, It will honor the value of config 'spark.rapids.memory.gpu.pooling.enabled'|None
 <a name="shuffle.enabled"></a>spark.rapids.shuffle.enabled|Enable or disable the RAPIDS Shuffle Manager at runtime. The [RAPIDS Shuffle Manager](additional-functionality/rapids-shuffle.md) must already be configured. When set to `false`, the built-in Spark shuffle will be used. |true
+<a name="shuffle.mode"></a>spark.rapids.shuffle.mode|RAPIDS Shuffle Manager mode. The default mode is "UCX", which has to be installed in the system. Consider setting to "CACHE_ONLY" if running with a single executor and UCX is not installed, for short-circuit cached shuffle (for testing purposes). Set to "MULTI_THREADED" for an experimental mode that uses a thread pool to speed up shuffle writes without needing UCX. Note: Changing this mode dynamically is not supported.|UCX
```

Comment:
nit we use MULTITHREADED in other configs, perhaps we should keep it consistent rather then having the _